GW2EIEvtcParser/EncounterLogic/Strikes/EndOfDragons/HarvestTemple.cs

```diff
@@ -496,10 +518,36 @@ internal override void EIEvtcParse(ulong gw2Build, FightData fightData, AgentDat
                         break;
                     case (int)ArcDPSEnums.TrashID.PushableVoidAmalgamate:
                     case (int)ArcDPSEnums.TrashID.KillableVoidAmalgamate:
+                        if (target.ID == (int)ArcDPSEnums.TrashID.PushableVoidAmalgamate)
+                        {
+                            pushableOrbAgents.Add(target.AgentItem.Agent);
+                        }
                         target.OverrideName("Heart " + (++purificationID));
                         break;
                 }
             }
+
+            // Overwrite target damage in push phases to show number of strike hits by players as the main metric of the phase
+            foreach (CombatItem cbt in combatData)
```

Comment:
Add a PlayerSrcAllHitsMechanic instead, checking evt.To species to match orbs. You can see it used in KC in order to check hits against the core.
Goota say, what you are doing here is a little bit too extreme for my taste xD

Reply:
This also hide actual damage done against the orb in the phase where it needs to be killed.

Reply:
Kinda agree it is a hack and didn't think about displaying it as mechanic. I did a quick rebase deleted commit you commented on and just added orb push as mechanic the way you recommended it. Still readable enough on graph so good for me.